pset-functions.js

```diff
@@ -31,6 +31,36 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+
+// Simulate blackjack hands and return the greatest valid
+// result of the winner (or winners in a draw)
+function blackJack(playerCardScore, dealerCardScore) {
+  // If both players exceed 21 return 0 - no winners
+  if (playerCardScore > 21 && dealerCardScore > 21) {
```

Comment:
@ksuitor nice use of a guard clause to prevent the rest of the function from executing in the event that both cards exceed 21